### ISSUE TITLE
Redirection to French content not working

### ISSUE THREAD

**Issue by agriffard** [aspnet/Docs.fr-fr#197]
When I am on https://docs.microsoft.com/fr-fr/aspnet/index?view=aspnetcore-2.2#pivot=core&panel=core_overview

If I click on the first 'Bien démarrer avec ASP.NET Core', the link is
https://docs.microsoft.com/aspnet/core/getting-started and it redirects me to https://docs.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-2.2&tabs=windows 

But as I am coming from an fr-fr page, it should redirect to https://docs.microsoft.com/fr-fr/aspnet/core/getting-started/?view=aspnetcore-2.2&tabs=windows instead

**Comment by LisandroSu** [comment]
Hello, @agriffard 
 
Thank you for your feedback.
 
We are actively reviewing your comments and will get back to you soon.

Kind regards,
Microsoft DOCS International Team

**Comment by LisandroSu** [comment]
Hi, @JasonCard 

Can you check this issue? Is the link doing this by design?

Thanks!
Lisandro

**Comment by JasonCard** [comment]
Hi @agriffard,

Thank you for your feedback on this.
This is currently known issue where the locale in URL is lost for articles that are redirected, and locale defaults to user’s locale setting in docs.microsoft.com (which is en-us by default).
We have a feature request to fix this behavior and it will be prioritized in upcoming planning.

Best regards,
Microsoft DOCS International Team
cc @LisandroSu

**Comment by CeciAc** [comment]
Hello, @agriffard 
 
Thank you for your feedback.
 
We are actively reviewing your comments and will get back to you soon.

Kind regards,
Microsoft DOCS International Team

**Comment by CeciAc** [comment]
Hi @JasonCard 
any updates on this one?
kind regards,

**Comment by JasonCard** [comment]
@CeciAc feature request is still in backlog. 
Pinged assignee, but still no ETA.

**Comment by CeciAc** [comment]
Hi @JasonCard 
is this still on process?

**Comment by JasonCard** [comment]
@CeciAc Yes, it's still in backlog, but I'm following it and will let you know when we have an ETA.
